src/routes/business/authorizedPerson.ts

```diff
@@ -0,0 +1,79 @@
+import type { Response } from "express";
+
+import generateID from "../../helpers/id";
+import uuid from "node-uuid";
+import { AuthorizedPerson } from "../../helpers/types";
+import { saveBusiness, getBusinessIdByAccount } from "../../db";
+import { RequestWithBusinessAndAccount } from "../../helpers/middlewares";
+
+export const createAuthorizedPerson = async (
+  req: RequestWithBusinessAndAccount,
+  res: Response
+) => {
+  const { business } = req;
+  const { account_id } = req.params;
+
+  const authorizedPersonId = req.body.authorized_person_id;
+  if (!authorizedPersonId) {
+    return res.status(400).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 400,
+          code: "missing_field",
+          title: "Missing Field",
+          detail: "Invalid request.",
+        },
+      ],
+    });
+  }
+
+  const businessId = await getBusinessIdByAccount({ id: account_id });
+  if (businessId !== business.id) {
+    return res.status(403).send({
+      errors: [
+        {
+          id: uuid.v4(),
+          status: 403,
+          code: "forbidden",
+          title: "Forbidden",
+          detail: "You are not authorized to perform this action.",
+        },
+      ],
+    });
+  }
+
+  try {
+    const authorizedPerson: AuthorizedPerson = {
+      id: generateID(),
+      authorized_person_id: authorizedPersonId,
+      business_id: business.id,
+      account_id,
+      valid_from: new Date().toISOString().split("T")[0],
+      valid_until: new Date(
```

Comment:
have you tested it on sandbox and solaris does return this 10 years interval?

Reply:
No, I assumed it based on the Solaris docs sample, I'll check it.

Reply:
OK, I've set it to null, but let's keep others, right now it's implemented and we can keep it.
changed by https://github.com/kontist/mock-solaris/pull/210/commits/b1ef2f042ec4f761953945fce2db2b68c0412d37.